cmd/kpod/ps.go

```diff
@@ -31,7 +31,6 @@ type psOptions struct {
 	noTrunc   bool
 	quiet     bool
 	size      bool
-	label     string
```

Comment:
@baude Should we keep this? Not sure

Reply:
FWIW, I'm not seeing it being used anywhere.  Looks to be safe to remove for now and readd if necessary.   @baude WDYT?

Reply:
I would say to leave it for now, just me though

Reply:
ok done, left it as it is!